### ISSUE TITLE
Username and password

### ISSUE THREAD

**Issue by newmohammedtakalah1994** [DrBrad/JTun2Socks#5]
Hello, does this app support username and password  for server socks

**Comment by DrBrad** [comment]
No, it currently doesn't support accounts, however you can modify the Proxy.java to accept accounts.

As I recall for socks server accounts it's only a byte change and the credentials.
